### PR TITLE
feat(releasing): Add release script to CDC

### PR DESCRIPTION
**Pull request by fpacifici**
We will have to cut releases. 

This now does not follow calver, but that is up for debate. CDC is not a core part of sentry, more of an external dependency (it is also not BSL @BYK  what do you think? ).

### CODE REVIEW

File: .craft.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+github:
+  owner: getsentry
+  repo: cdc
+# TODO: Add a changelog
+changelogPolicy: none
+
+artifactProvider:
+  name: none
+statusProvider:
+  name: github
```
Comment (BYK) [inline .craft.yml:12]:
```suggestion
```

GitHub is the default after `0.21.0`.

File: .craft.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+minVersion: "0.20.0"
```
Comment (BYK) [inline .craft.yml:1]:
```suggestion
minVersion: "0.21.1"
```

**Review by BYK — APPROVED** [review]
>We will have to cut releases.

Can you elaborate on why?

>This now does not follow calver, but that is up for debate. CDC is not a core part of sentry, more of an external dependency (it is also not BSL @BYK what do you think? ).

If this is a project like Snuba, specific to Sentry and needs to go in lock-step, it should use CalVer and BSL. If it is a project that can be used outside of Sentry, like Symbolicator, then we should use Apache-2 and SemVer.

**Comment by BYK** [comment]
>We could reference a specific commit otherwise.

The approach we have taken with Symbolicator is to take the latest published release instead of latest master commit SHA.

**Comment by fpacifici** [comment]
> The approach we have taken with Symbolicator is to take the latest published release instead of latest master commit SHA.

Right, so we do have to cut and publish releases for that, or am I missing something?

**Comment by BYK** [comment]
>Right, so we do have to cut and publish releases for that, or am I missing something?

Yes we do.

**Pull request merged**